### PR TITLE
fix: image desktop

### LINKED ISSUE #15560: Can't open image sent from desktop
# Bug Report

#### Expected behavior
can open image on mobile

#### Actual behavior
IMAGE 2023-04-03 11:29:55

### Reproduction

1) Desktop: send image or album
2) Mobile: open image

### Additional Information

- Status version: nightly 
- Operating System: Android (crash), iOS (cannot be opened)
- Logs: [Status-debug-logs.zip](https://github.com/status-im/status-mobile/files/11136812/Status-debug-logs.zip)

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/15560

### CODE REVIEW

**Review by ibrkhalil — APPROVED** [review]
Defaulting to a squared image .. Okay

File: src/status_im2/contexts/chat/lightbox/zoomable_image/view.cljs [hunk 1]
```diff
@@ -216,8 +216,8 @@
            curr-orientation                        (or (rf/sub [:lightbox/orientation])
                                                        orientation/portrait)
            portrait?                               (= curr-orientation orientation/portrait)
-           dimensions                              (utils/get-dimensions image-width
-                                                                         image-height
+           dimensions                              (utils/get-dimensions (or image-width 1000)
```
Comment (J-Son89) [inline src/status_im2/contexts/chat/lightbox/zoomable_image/view.cljs]: should `1000` be a constant? or at least a local named variable?

**Review by J-Son89 — APPROVED** [review]

**Review by ilmotta — APPROVED** [review]

File: src/status_im2/contexts/chat/messages/composer/controls/view.cljs [hunk 2]
```diff
@@ -40,12 +40,12 @@
...
 
 (defn image-button
-  []
+  [bottom-inset]
   [quo/button
    {:on-press (fn []
                 (permissions/request-permissions
                  {:permissions [:read-external-storage :write-external-storage]
-                  :on-allowed  #(rf/dispatch [:open-modal :photo-selector])
+                  :on-allowed  #(rf/dispatch [:open-modal :photo-selector {:bottom-inset bottom-inset}])
```
Comment (Parveshdhull) [inline src/status_im2/contexts/chat/messages/composer/controls/view.cljs:48]: Hi @OmarBasem, why we are making these changes in PR
1. Changes doesn't look related to issue itself
2. It looks very confusing variable name is also bottom-inset, it also has a key with same name

File: src/status_im2/contexts/chat/photo_selector/view.cljs [hunk 3]
```diff
@@ -25,13 +24,13 @@
...
 (defn bottom-gradient
-  [selected-images insets selected]
+  [selected-images bottom-inset selected]
   (when (or (seq @selected) (seq selected-images))
     [linear-gradient/linear-gradient
      {:colors [:black :transparent]
       :start  {:x 0 :y 1}
       :end    {:x 0 :y 0}
-      :style  (style/gradient-container insets)}
+      :style  (style/gradient-container bottom-inset)}
```
Comment (Parveshdhull) [inline src/status_im2/contexts/chat/photo_selector/view.cljs:33]: For example here, in insets I know it hold :top, :bottom keys etc.
I don't know what bottom-inset has. Is it value itself or a map with key-values

File: src/status_im2/contexts/chat/messages/composer/controls/view.cljs (hunk 2 above)
Comment (OmarBasem) [reply src/status_im2/contexts/chat/messages/composer/controls/view.cljs:48]: @Parveshdhull this is reverting some changes from that PR https://github.com/status-im/status-mobile/pull/15467

For now, we need to use the insets from screen-params. Not doing so causes the photo selector on Android to be unusable (the bottom gradient cover the whole screen due to incorrect insets).

There is an open issue to change the way we use insets: https://github.com/status-im/status-mobile/issues/15535

**Pull request merged**